### PR TITLE
Control theme

### PR DESCRIPTION
**Pull request by jinek**
There are number of issues, but seems mostly come from other changes.
I will re-test the application entirely and will create bugs whether they come from this PR or others.

### CODE REVIEW

File: src/Consolonia.Themes.TurboVision/Templates/Controls/MenuItem.axaml [hunk 1]
```diff
@@ -97,89 +97,37 @@
...
-    <Style Selector="MenuItem:disabled">
-        <Setter Property="Foreground"
-                Value="{DynamicResource ThemeNoDisturbBrush}" />
-    </Style>
-</Styles>
\ No newline at end of file
+        <Style Selector="^:disabled">
+            <Setter Property="Foreground"
+                    Value="{DynamicResource ThemeNoDisturbBrush}" />
+        </Style>
```
[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Themes.TurboVision/Templates/Controls/MenuItem.axaml:131]: We just want to be touch-friendly. Keeping all the same for consistency :D

File: src/Consolonia.Themes.TurboVision/Templates/Controls/DataGrid.axaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-        xmlns:drawing="clr-namespace:Consolonia.Core.Drawing;assembly=Consolonia.Core">
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
+            xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+            xmlns:drawing="clr-namespace:Consolonia.Core.Drawing;assembly=Consolonia.Core"
+            xmlns:helpers="clr-namespace:Consolonia.Themes.TurboVision.Templates.Controls.Helpers;assembly=Consolonia.Themes.TurboVision">
     <!--TODO: Validation and Focus-->
 
-    <Styles.Resources>
-        <Thickness x:Key="DataGridTextColumnCellTextBlockMargin">1,0</Thickness>
-    </Styles.Resources>
```
[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Themes.TurboVision/Templates/Controls/DataGrid.axaml:7]: Yes, feel free to implement it.

File: src/Consolonia.Themes.TurboVision/Templates/Controls/MenuItem.axaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Themes.TurboVision/Templates/Controls/MenuItem.axaml:131]: No, that was a joke. Remove that from your notes. We do not react to mouse-over events coz of performance reasons and simplicity. Touch-screen support is in the future only.

File: src/Consolonia.Themes.TurboVision/Templates/Controls/DataGrid.axaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Themes.TurboVision/Templates/Controls/DataGrid.axaml:7]: There is no editing there. Forget it

File: src/Consolonia.Themes.TurboVision/Themes/TurboVisionDark/TurboVisionDarkTheme.cs [hunk 3]
```diff
@@ -1,20 +1,14 @@
...
 namespace Consolonia.Themes.TurboVision.Themes.TurboVisionDark
 {
-    // ReSharper disable once UnusedType.Global
-    public class TurboVisionDarkTheme : ResourceIncludeBase
+    public class TurboVisionDarkTheme : Styles
     {
-        public TurboVisionDarkTheme(Uri baseUri) : base(baseUri)
+        public TurboVisionDarkTheme(IServiceProvider sp = null)
         {
+            AvaloniaXamlLoader.Load(sp, this);
```
Comment (tomlm) [inline src/Consolonia.Themes.TurboVision/Themes/TurboVisionDark/TurboVisionDarkTheme.cs:11]: ooooh. I didn't know there was a IServiceProvider signature for this. That's cool.

**Review by tomlm — APPROVED** [review]
yay. I didn't look super closely at this since there is not a lot of code. Did you have to make  lot of changes to what I did or just a bunch of tweaks?

File: src/Consolonia.Themes.TurboVision/Themes/TurboVisionDark/TurboVisionDarkTheme.cs (hunk 3 above)
Comment (jinek) [reply src/Consolonia.Themes.TurboVision/Themes/TurboVisionDark/TurboVisionDarkTheme.cs:11]: I copypasted most of it from avalonia. Im even not sure how everything is supposed to work. I'm lost between styles, themes, and resource dictionaries.

**Comment by jinek** [comment]
> yay. I didn't look super closely at this since there is not a lot of code. Did you have to make  lot of changes to what I did or just a bunch of tweaks?

Mostly tweeks. The weirdest thing was the binding stopped working, i have replaced it with another one. I couldn't debug it quick, looked like avalonia issue.

**Pull request merged**